blog.md

```diff
@@ -1,21 +1,38 @@
 # Simplygon in Azure
 
-I wrote a previous blog post about Simplygon a few years ago and the idea of optimising 3D meshes to enable running on a low powered device such as a HoloLens or mobile phone has cropped up on a number of occasions since. The content in the more recent projects have the additional requirement of being loaded into an app dynamically when requested by the app user. Most projects designed to run across these kinds of mobile devices usually need to address the question of whether to render content on a server or locally on the device. Sometimes that choice will be obvious and other times it will be more complicated and represent a trade-off between many competing factors.
+![Simplygon](./images/main-simplygon.png)
 
-> On one hand if you have large 3D models with dense geometry consisting of millions of triangles where you need to see all of the fine details and the content isn't well suited for optimisation then server-side rendering might be the right choice for you. You might need to account for the infrastructure and maintenance work required to set up a server-side rendering solution and you also will need to understand the costs associated with running cloud services for this kind of a solution. You may need to scale across millions of users or you may only need to spin up and down services at a much smaller scale. If the latter applies then Azure Remote Rendering would be a suitable consideration. If the former, then maybe 3D Streaming Toolkit or a custom client/server rendering solution might be more cost-effective.
+I wrote a [previous blog post](http://peted.azurewebsites.net/hololensoptimising-with-simplygon/) about Simplygon a few years ago and the idea of optimising 3D meshes to enable running on a low powered device such as a HoloLens or mobile phone has cropped up on a number of occasions since. The content in the more recent projects have the additional requirement of being loaded into an app dynamically when requested by the app user. Most projects designed to run across these kinds of mobile devices usually need to address the question of whether to render content on a server or locally on the device. Sometimes that choice will be obvious and other times it will be more complicated and represent a trade-off between many competing factors.

```

Comment:
I feel like the first sentence could be two sentences 😅

Reply:
Seems ok to me - happy to change if you can propose what the sentences should be?

Reply:
In a previous blog post I wrote about using Simplygon to optimize 3D meshes for low powered devices such as HoloLens or mobiles phones. Ever since, this idea cropped up a number of times with requirements like dynamically loading 3D content tailored to the user's device.

something like that?